### ISSUE TITLE
Set all option but i can't see anything

### ISSUE THREAD

**Issue by Z4RR0** [aukedejong/lovelace-windrose-card#81]
I i'm try to set all option the entity working good but in the editor or the dashboard i can't see anything also the title in the card

I'm wrong something?

i'm on last home assistant and version of the integration

**Comment by aukedejong** [comment]
Can you share your card configuration?

**Comment by aukedejong** [comment]
Used your configuration, replaced the entity names, works here.
You are using only 3 hours history, maybe increasing the hours_to_show?

Also, the max_width property is not used anymore.  I'm curious, which version of the card are you using?

**Comment by aukedejong** [comment]
When you create a new card, it generated an configuration. Does it has the max_width in there?  It should not.

Do you see any errors in the browsers console?  Press F12.

**Comment by Z4RR0** [comment]
i try different time i see this in the F12 console

Uncaught (in promise) Error: Entity sensor.direzione_vento_gradi_meteo did
not return data, is this the correct entity name?
    checkLoadedData
http://home/hacsfiles/lovelace-windrose-card/windrose-card.js?hacstag=591270696190:34
    getMeasurements
http://home/hacsfiles/lovelace-windrose-card/windrose-card.js?hacstag=591270696190:34
windrose-card.js:34:141851

but my entity working correct i try and entity that report a letter (like
N....) or this with degree

Correct? or i need a template?

Il giorno lun 7 ott 2024 alle ore 19:31 Auke de Jong <
***@***.***> ha scritto:

**Comment by Z4RR0** [comment]
[image: immagine.png]
the entity working but on the rose no. But Why?=??  Thnk you for the support

Il giorno mer 9 ott 2024 alle ore 12:16 Mauro Miolli ***@***.***>
ha scritto:

**Comment by aukedejong** [comment]
Strange.
Did you try to use statistics data?

**Comment by Z4RR0** [comment]
Yes, but nothing change. a question my entities are from a json so i made a template for ever single sensor  but nothing change 
so there are a specific type of sensor entities? like number o string or sometigh other

i try another card "aka compass card and working " but i love rose compass card. i need to want to solve!

thank's for the support

**Comment by il77781** [comment]
@Z4RR0, just to check: do the entities you use have the `state_class: measurement` attribute?

**Comment by angel41671** [comment]
Hi there.
I had the same issue. I solved it by setting the speed units from "auto" to "kph":
speed_unit: kph

I hope it works also for you!

**Comment by il77781** [comment]
@angel41671, apparently, the reasons can be very different...

**Comment by Z4RR0** [comment]
I try also a new instance of ha but nothing i cant understand why

I try to create a helper if the type of entity are incompatible but nothing
work 🥲

Il Sab 8 Feb 2025, 23:38 il77781 ***@***.***> ha scritto:

**Comment by aukedejong** [comment]
What output do you get in the browsers console (F12) when you set the log_level config to TRACE?

`log_level: TRACE`

Can you post the complete log here?

**Comment by Z4RR0** [comment]
so i add in wind_rose card section option: log_level: TRACE ? right? then
F12 to browser console?

Il giorno dom 9 feb 2025 alle ore 11:07 Auke de Jong <
***@***.***> ha scritto:

**Comment by aukedejong** [comment]
You add that to the windrose card configuration.

```
type: custom:windrose-card
title: Windrose card
log_level: TRACE

other configs...
```

Wind the windrose on the dashbaord, refresh the page with the browser console open.

**Comment by Z4RR0** [comment]
Thank's for the tips so i made it and in the console i see this message:

WR TRACE EntityStatesProcessor: Updated compass values: false  undefined
windrose-card.js:50:137515
WR TRACE EntityStatesProcessor: Updated wind values: false  undefined
windrose-card.js:50:137515

so i think i have a problem with my entity?

if i want to try to set manual entity and give a fix value to made working
so i can understand why windrose don't accept my value?

Thank's a lot for the support

Il giorno dom 9 feb 2025 alle ore 19:40 Auke de Jong <
***@***.***> ha scritto:

**Comment by Z4RR0** [comment]
Finally WORK! i find my issue that don't show anything.... A simple mistake!!

@aukedejong if the unit of speed wind entity was "Kmh" and not "kmh" i don't see anything.

So i changhe unit from Kmh to kmh and work!

![Image](https://github.com/user-attachments/assets/6274b816-ac75-4e6d-9922-0a3e74141694)

**Comment by aukedejong** [comment]
Nice, good work.
I will change the card to be case insensitive, so somebody else in the future won't run into this.

Thanks, this will improve the card.

**Comment by Z4RR0** [comment]
After million of test 😅

Il Sab 15 Mar 2025, 10:46 Auke de Jong ***@***.***> ha
scritto:
